pkg/cmd/workspace/util/branch_wizard.go

```diff
@@ -17,12 +17,13 @@ import (
 )
 
 type BranchWizardConfig struct {
-	ApiClient    *apiclient.APIClient
-	ProviderId   string
-	NamespaceId  string
-	Namespace    string
-	ChosenRepo   *apiclient.GitRepository
-	ProjectOrder int
+	ApiClient       *apiclient.APIClient
+	ProviderId      string
+	NamespaceId     string
+	Namespace       string
+	ChosenRepo      *apiclient.GitRepository
+	ProjectOrder    int
+	GitProviderName string
```

Comment:
Why did you add this new property? Why not use `ProviderId`?

Reply:
I though I already said it multiple times... 😅 

`ProviderId` **must** be the same value that `ProviderId` is on the server.

The `Id` is the UUID.

Reply:
Please stop rushing us.

Reply:
1. `GitConfigId` should be `GitProviderConfigId`
2. `GitProviderName` should be `ProviderId`